eslint-config-dabapps-react-native/package.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
if we remove all these package.jsons, we can make it much cleaner. So in the project we just do 
`
"extends": {
    "eslint-config-dabapps/react-native"
}
`which is so much nicer!